packages/glimmer-runtime/tests/updating-test.ts

```diff
@@ -547,6 +547,28 @@ test("class attribute w/ concat follow the normal dirtying rules", function() {
   equalTokens(root, "<div class='hello world'>hello</div>");
 });
 
+test("attribute is removed if the binding becomes falsy", function() {
```

Comment:
I think this only applies to `class` right?  If that is true, can you update this to say "class attribute is removed if...."?